OrbitGl/TimeGraph.cpp

```diff
@@ -621,30 +621,37 @@ std::string GetTimeString(const TimerInfo& timer_a, const TimerInfo& timer_b) {
   return colors[index % kNumColors];
 }
 
-void DrawIteratorBox(GlCanvas* canvas, Vec2 pos, Vec2 size, const Color& color,
+}  // namespace
+
+void TimeGraph::DrawIteratorBox(GlCanvas* canvas, Vec2 pos, Vec2 size, const Color& color,
                      const std::string& label, const std::string& time, float text_y) {
   Box box(pos, size, GlCanvas::kZValueOverlay);
   canvas->GetBatcher()->AddBox(box, color);
 
   std::string text = absl::StrFormat("%s: %s", label, time);
 
   constexpr const float kAdditionalSpaceForLine = 10.f;
-  constexpr const float kLeftOffset = 5.f;
+  const Color kWhite(255, 255, 255, 255);
+  const Color kBlack(0, 0, 0, 255);
 
   float max_size = size[0];
-  canvas->GetTextRenderer().AddTextTrailingCharsPrioritized(
-      text.c_str(), pos[0] + kLeftOffset, text_y + kAdditionalSpaceForLine, GlCanvas::kZValueText,
-      Color(255, 255, 255, 255), time.length(), GParams.font_size, max_size);
+  float text_width = canvas->GetTextRenderer().AddTextTrailingCharsPrioritized(
+      text.c_str(), pos[0], text_y + kAdditionalSpaceForLine, GlCanvas::kZValueText,
+      kBlack, time.length(), GParams.font_size, max_size);
 
   constexpr const float kOffsetBelowText = kAdditionalSpaceForLine / 2.f;
+  Vec2 white_box_size(text_width, GetTextBoxHeight());
+  Vec2 white_box_position(pos[0], text_y + kOffsetBelowText);
```

Comment:
TextBoxHeight will scale with the vertical zooming, but either GParams.font_size nor kOffsetBelowText not.

This is what you see after scaling the text in manual instrumentation variables:
![Screen Shot 2020-10-01 at 11 50 22](https://user-images.githubusercontent.com/8610429/94795427-9822b800-03dd-11eb-8884-ab1a30409515.png)

This is what you see after scaling with the iterator:
![Screen Shot 2020-10-01 at 11 49 38](https://user-images.githubusercontent.com/8610429/94795530-bdafc180-03dd-11eb-8464-3111ca00c6e1.png)


There are 2 possible solutions:
1- Don't scale the text box height (modifying the method and adding a new one GetScalableTextBoxHeight or whatever)
2- Scale everything by modifying:
GParams.font_size -> GetFontSize()
kOffsetBelowText -> GetTextOffset
kAdditionalSpaceForLine -> GetTextOffset x 2

Reply:
Done.